roles/ovirt-permissions/README.md

```diff
@@ -12,17 +12,17 @@ Requirements
 Role Variables
 --------------
 
-The item in `permissions` list can contain following parameters:
+The `permissions` list can contain following parameters:
 
 | Name          | Default value  | Description                |
 |---------------|----------------|----------------------------|
-| state         | present        | State of the permission    |
-| user_name     | UNDEF          | User name of user to manage permission |
-| group_name    | UNDEF          | Name of group to manage permission |
-| authz_name    | UNDEF          | Name of the authorization provider of the group or user |
-| role          | UNDEF          | Role to be assigned to user or group |
-| object_type   | UNDEF          | The object type which should be used to assign permission |
-| object_name   | UNDEF          | Name of the object where permission should be assigned |
+| state         | present        | Specifies whether the state of the permission is `present` or `absent`.    |
+| user_name     | UNDEF          | The user to manage the permission. |
+| group_name    | UNDEF          | Name of group to manage the permission. |
```

Comment:
I'll make this update. I wasn't sure if we meant, "The user to manage the permission.", or "The user to manage the permission for."